generator/cli/index.ts

```diff
@@ -17,12 +19,19 @@ main().catch((error) => {
 async function main(): Promise<void> {
   const dirName = 'templates-repo'
   const templatesRepoDir = Path.join(__dirname, '../..', dirName)
-
+  const migrationSqlOutputDir = Path.join(__dirname, '../../src/generatedMigrations')
   if (args['--download-templates-repo']) {
     //eslint-disable-next-line
     await downloadTemplatesRepo({ dir: templatesRepoDir })
   }
 
+  if (args['--generate-migration-sql']) {
+    await generateMigrationSql({
+      templatesRepoDir,
```

Comment:
If the templates repo dir is not available then we cannot proceed. Can you do a simple validation check that at least the dir exists and is not empty or something?